src/app/clusters/general-diagnostics-server/tests/TestGeneralDiagnosticsCluster.cpp

```diff
@@ -108,9 +140,9 @@ TEST_F(TestGeneralDiagnosticsCluster, AttributesTest)
                   GeneralDiagnostics::Commands::TimeSnapshot::kMetadataEntry.GetInvokePrivilege());
 
         // Check required generated commands are present
-        ReadOnlyBufferBuilder<chip::CommandId> generatedCommandsBuilder;
+        ReadOnlyBufferBuilder<CommandId> generatedCommandsBuilder;
```

Comment:
we should probably switch these to use the helpers in https://github.com/project-chip/connectedhomeip/blob/master/src/app/clusters/testing/ValidateGlobalAttributes.h#L65